src/main/java/com/rhombus/api/examples/AddRemoveLabels.java

```diff
@@ -0,0 +1,126 @@
+package com.rhombus.api.examples;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.rhombus.ApiClient;
+import com.rhombus.sdk.FaceWebserviceApi;
+import com.rhombus.sdk.domain.FaceAddFaceLabelWSRequest;
+import com.rhombus.sdk.domain.FaceRemoveFaceLabelWSRequest;
+import org.apache.commons.cli.*;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJaxbJsonProvider;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+import javax.ws.rs.client.ClientBuilder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class AddRemoveLabels
+{
+    private static ApiClient _apiClient;
+    private static FaceWebserviceApi _faceWebservice;
+
+    public static void main(String[] args) throws Exception
+    {
+        final Options options = new Options();
+        options.addRequiredOption("a", "apikey", true, "API Key");
+        options.addRequiredOption("c", "choice", true, "Add or remove a label");
+        options.addRequiredOption("l", "label", true, "Label name");
+        options.addRequiredOption("n", "names", true, "Names of people to add or remove labels from (ex: name, name, name)");
+
+        final CommandLine commandLine;
+        try {
+            commandLine = new DefaultParser().parse(options, args);
+        } catch (ParseException e) {
+            System.err.println(e.getMessage());
+
+            new HelpFormatter().printHelp(
+                    "java -cp build/libs/rhombus-api-examples-all.jar com.rhombus.api.examples.AddRemoveLabels",
```

Comment:
no need to put `build/libs` before the java jar since depending on how a person has compiled, the location of the jar may be different